spec/requests/api/allocation_events_controller_cancel_spec.rb

```diff
@@ -8,9 +8,7 @@
   describe 'POST /allocations/:id/cancel' do
     let(:schema) { load_yaml_schema('post_allocation_cancel_responses.yaml') }
 
-    let(:supplier) { create(:supplier) }
-    let(:application) { create(:application, owner_id: supplier.id) }
-    let(:access_token) { create(:access_token, application: application).token }
+    let(:access_token) { 'spoofed-token' }
```

Comment:
I think we may use this snippet everywhere - could you move it to a context so it can be included (`include 'spoof token'`)
```
let(:access_token) { 'spoofed-token' }
 let(:headers) { { 'CONTENT_TYPE': content_type, 'Authorization': "Bearer #{access_token}" } }
let(:content_type) { ApiController::CONTENT_TYPE }
```

Reply:
There's not enough consistency in the tests yet - there are lots of subtle variations on this and I didn't want to introduce even more noise. Happy to look at that again in a future PR though.

Reply:
ok 👍